composer.json

```diff
@@ -9,9 +9,10 @@
   ],
   "homepage": "https://www.opus-repository.org",
   "require": {
-    "opus4-repo/opus4-common": "^4.7.1",
-    "opus4-repo/framework": "^4.7.1",
-    "iio/libmergepdf": "^4.0"
+    "opus4-repo/opus4-common": "dev-master",
```

Comment:
War das notwendig? Ich glaube **opus4-pdf** (main) hat sich seit dem Release von 4.7.1 noch nicht weiter verändert. Wenn es möglich bleiben soll, eine neue Version des Deckblatt-Supports zu 4.7.1 Installationen hinzuzufügen, dann müssen die Abhängigkeiten zurück gesetzt werden. Für Integrationstests müssen dann mit dem MASTER Branch (also 4.7.1) in Application gemacht werden. Für die 4.7.2 ändert sich viel Code. Wenn das PDF dafür jetzt schon angepasst wird, dann ist die Veröffentlichung erst mit dem Release von 4.7.2 möglich.

Reply:
Ah ja richtig, mein Fehler. Dies war ein Missverständnis, das ändere ich wieder zurück.

Reply:
Keine Framework-Abhängigkeit? Spötestens wenn die Metadaten ausgelesen werden sollen stimmt das nicht mehr, oder?